sdk/nodejs/index.js

```diff
@@ -65,13 +72,15 @@ class Picovoice {
     this.inferenceCallback = inferenceCallback;
 
     this.porcupine = new Porcupine(
+      accessKey,
       [keywordPath],
       [porcupineSensitivity],
       porcupineModelPath,
       porcupineLibraryPath
     );
 
     this.rhino = new Rhino(
+      accessKey,
```

Comment:
`require_endpoint`? I think we need to expose this in the Picovoice API unfortunately.